conans/client/command.py

```diff
@@ -553,6 +556,9 @@ def info(self, *args):
 
             if args.graph:
                 self._outputer.info_graph(args.graph, deps_graph, get_cwd())
+            elif args.json:
```

Comment:
Make it compatible to use `graph` and `json` outputs? Raise an error if both are requested?

Reply:
probably using this would be sufficient https://graphviz.gitlab.io/_pages/doc/info/output.html#d:json

Reply:
This output is not the JSON of the graphviz, it contains much more information. It would be interesting to build the `.dot` file using the JSON information (instead of passing the graph data), but that is about a refactor of the outputs, not for this PR.